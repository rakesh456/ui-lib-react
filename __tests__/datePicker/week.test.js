import DEFAULT_OPTIONS from '../../src/utils/constants';
import renderer from 'react-test-renderer';
import React from 'react';

test('Week component renders the week element correctly', () => {
  const options = DEFAULT_OPTIONS;
  const rendered = renderer.create(
    <week options={options} />
  );
  expect(rendered.toJSON()).toMatchSnapshot();
});