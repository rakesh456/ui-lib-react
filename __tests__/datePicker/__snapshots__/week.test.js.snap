// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Week component renders the week element correctly 1`] = `<week />`;
